doozerlib/cli/images_streams.py

```diff
@@ -1189,7 +1212,7 @@ def check_if_upstream_image_exists(upstream_image):
                     yellow_print(f'Unable to add labels to {existing_pr.html_url}: {str(pr_e)}')
 
                 pr_url = existing_pr.html_url
-                pr_links[dgk] = pr_url
+                pr_links[dgk] = existing_pr
```

Comment:
pr_url is no longer used.

Reply:
Also pr_links should be renamed since it no longer means "PR links".

Reply:
update pr_links to pr_dgk_map